Backend/Controllers/DeckController.cs

```diff
@@ -0,0 +1,58 @@
+﻿using Backend.Services;
+using Domain.Entities;
+
+namespace Backend.Controllers
+{
+    public static class DeckController
+    {
+
+        public static void MapDeckEndpoint(this WebApplication app)
+        {
+            //Get all decks
+            app.MapGet("/decks", (IDeckService deckService) =>
+            {
+                return deckService.GetUserDecks();
```

Comment:
Vi skal overveje om vi skal parse et userId med her, hvis vi har user decks. Vi kan godt tage det fra vores token, men vores service skal nok tage et ID ind som parameter

Reply:
Cannot be done, because of missing JWT decryption of le token